test/e2e/tests/test_notebook_instance_lifecycle.py

```diff
@@ -119,19 +128,18 @@ def test_CreateUpdateDeleteNotebookLifecycleConfig(
 
         resource = k8s.wait_resource_consumed_by_controller(reference)
         assert resource is not None
-        assert (self.wait_until_update(reference, lastModifiedTime) == True)
+        assert self.wait_until_update(reference, lastModifiedTime) == True
 
         # Verifying that an update was successful
         notebook_instance_lfc_desc = get_notebook_instance_lifecycle_config(
             notebook_instance_lfc_name
         )
-        assert (
-            notebook_instance_lfc_desc["OnStart"][0]["Content"]
-            == update_content
-        )
+        assert notebook_instance_lfc_desc["OnStart"][0]["Content"] == update_content
 
         # Deleting the resource
-        _, deleted = k8s.delete_custom_resource(reference, 10, 30)
+        _, deleted = k8s.delete_custom_resource(
+            reference, DELETE_WAIT_PERIOD, DELETE_WAIT_LENGTH
```

Comment:
afaik, its commonly referred to as `period_length` not` wait_length`

Reply:
correct, wait_period and period_length
also here
```
wait_until_update(
        self, reference, previous_modified_time, wait_period=30, wait_time=5
    )
```